pages/about.html

```diff
@@ -0,0 +1,203 @@
+            Please contact us if you have any further questions about web school!
+          </h2>
+          <a href="#" class="text-dark text-decoration-underline">
+            webscholl@gmail.com &raquo;
+          </a>
+        </div>
+      </div>
+      <div class="description mb-5 text-center">
+        <h2 class="h3">
+          What Is Web School?
+        </h2>
+        <p class="m-auto mb-5">
+          Are you tired of wasting your time and searching for useless courses and lost a lot of time? 
+          With Web School Academy, you will learn in the right way and with detailed academic explanations 
+          that explain everything in boring and smooth detail with a promise to understand everything 
+          no matter how difficult it is.
+          We exist 5 years ago. There are a lot of students who graduated from our school. And built their career.
+          And some of them have their own companies. Join now its time to make your success
+        </p>
+        <img src="../imgs/logo1.jpg" alt="logo" class="img-fluid"/>
+      </div>
+    </section>
+
+    <section class="past text-center mb-5">
+      <h2 class="title">
+        See the past of web School
+      </h2>
+      <p>
+        Take a look at the last 2 years of our work
+      </p>
+      <div class="last-years d-flex justify-content-evenly align-items-center flex-wrap">  
+        <div class="years p-5 text-white mt-2">
+          <div class="overlay"></div>
+          <span class="h1 fw-bold">
+            2019
+          </span>
+          <p>
+            Web school at 2019 in Egypt
+          </p>
+        </div>
+        <div class="years p-5 text-white mt-2">
+          <div class="overlay"></div>
+          <span class="h1 fw-bold">
+            2020
+          </span>
+          <p>
+            Web school at 2020 in US
+          </p>
+        </div>
+      </div>
+    </section>
+    <!-- Start Partner -->
+    <section class="partner container-fluid py-4 mb-5  d-lg-none">
+      <h2 class="title text-center pt-3 mb-5">
+        Partner
+      </h2>
+      <div class="d-flex flex-wrap justify-content-around align-items-center">
+        <div class="img-container mt-2">
+          <img src="../imgs/mozilla.png" alt="mozilla" class="w-100"/>
+        </div>
+        <div class="img-container mt-2">
+          <img src="../imgs/line-friends.png" alt="line-friends" class="w-100"/>
+        </div>
+        <div class="img-container mt-2">
+          <img src="../imgs/googlekorea.png" alt="googlekorea" class="w-100"/>
+        </div>
+        <div class="img-container mt-2">
+          <img src="../imgs/airbnb.png" alt="airbnb" class="w-100"/>
+        </div>
+      </div>
+    </section>
+     <!-- Start Footer -->
+     <footer class="container-fluid py-2 bg-dark text-white">
+      <div class="row justify-content-sm-around justify-content-center align-items-center">
+        <img src="../imgs/logo.jpg" alt="logo" class="col-lg-3 col-sm-3 col-7 my-4">
```

Comment:
- [ ] This image is too big, please reduce it and follow the original design. Check the layout too.

Your design:

![imagen](https://user-images.githubusercontent.com/60631456/132546193-25355057-4084-412f-a4eb-6a7750aed049.png)

Original design:

![imagen](https://user-images.githubusercontent.com/60631456/132546253-9dc2e3cc-2307-4689-9ff0-99a6d6cb1739.png)